secure-area-test-app/src/main/java/com/android/identity/secure_area_test_app/ui/theme/Theme.kt

```diff
@@ -0,0 +1,70 @@
+package com.android.identity.secure_area_test_app.ui.theme
```

Comment:
do files like this (+ Color.kt, Type.kt) need copyright at the top?

Reply:
Probably can't hurt. Added.